crates/vm/levm/src/vm.rs

```diff
@@ -910,7 +910,9 @@ impl VM {
             return Ok(OpcodeSuccess::Continue);
         }
 
-        current_call_frame.sub_return_data_offset = ret_offset;
+        current_call_frame.sub_return_data_offset = ret_offset
+            .try_into()
+            .map_err(|_err| VMError::VeryLargeNumber)?;
```

Comment:
We shouldn't return an error here if `size = 0`

Reply:
Done in [805c96b](https://github.com/lambdaclass/ethrex/pull/1458/commits/805c96bb7e5e1a0e6edc01c108c9f17f915d03fd), ended up changing the CallFrame to use U256 type in returndata offset